cmd/web3/main.go

```diff
@@ -170,6 +170,35 @@ func main() {
 				GetAddressDetails(ctx, network, c.Args().First(), privateKey)
 			},
 		},
+		{
+			Name:  "balance",
+			Usage: "Retrieve token balance",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "address",
+					Usage: "Address to get balance for",
+				},
+				cli.StringFlag{
+					Name:  "contract",
+					Usage: "ERC-20 contract to use to get balance. Use either this or token flag",
+				},
+				cli.StringFlag{
+					Name:   "private-key,pk",
+					Usage:  "Private key",
+					EnvVar: pkVarName,
+				},
+				cli.StringFlag{
+					Name:  "token,coin",
+					Usage: "Token to check, must be one of the tokens supported in this tool",
+					Value: "GO",
+				},
```

Comment:
What is the purpose of token/coin? Is this intended to look up a contract for a given symbol? That seems messy since they don't have to be unique.

Reply:
All the major tokens are unique, on mainnets anyways. That is what this is for. If you want to look at a particular contract that isn't the official one, you'd use the other flags.